apps/web/app/(routes)/users/page.tsx

```
@@ -3,41 +3,51 @@
-  // const [users, setUsers] = useState([]);
-  // Using dummy data instead of fetching from an API
-  const users = dummyUsers;
+  const API_ENDPOINT = process.env.NODE_ENV == "development" ? "http://localhost:3001/v1/user/public" : "https://api.indexflow.site/v1/user/public";
+
+  const [users, setUsers] = useState<User[]>([]);
+  useEffect(() => {
+    async function fetchServers() {
+      const data = await fetch(API_ENDPOINT);
+      const response = await data.json();
+      console.log("users", response);
+      setUsers([...response?.users, ...users]);
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)